requirements.txt

```diff
@@ -1,3 +1,3 @@
-pika<1.0
+pika>=1.0
```

Comment:
Does Pip pinning work in the same way I'm used to? E.g:

```
pika=*
```
☝️ Installs the latest

```
pika>=1.0
```
☝️ Installs the latest 1.x version.

Is that the same here, or will this literally install the latest version above or equal to 1.0?

Reply:
The >= is there because (and I'm pretty sure this is true) packages should be reasonably easy going about what is required.  The minimum that is needed for this to work is 1.0, so we might as well reflect that in the requirements

Reply:
Also, and it allows errors to be shown when you try and use it without the right version.  If something using this packages was using pika==0.13.0, then the first example would try and install the latest that it can (in this case 0.13.0 because it's explictly defined) then break because the interface changed.

Having the >=1.0 means that if something using this package tries to pin pika to 0.13.0, there will be a message like
`ERROR: sdc-rabbit 1.3.1 has requirement pika>=1.0, but you'll have pika 0.13.0 which is incompatible.`